compiler/rustc_hir_analysis/src/check/wfcheck.rs

```diff
@@ -961,13 +961,22 @@ fn check_param_wf(tcx: TyCtxt<'_>, param: &hir::GenericParam<'_>) -> Result<(),
                         hir_ty.span,
                         "using raw pointers as const generic parameters is forbidden",
                     ),
-                    _ => tcx.dcx().struct_span_err(
-                        hir_ty.span,
-                        format!("`{}` is forbidden as the type of a const generic parameter", ty),
-                    ),
+                    _ => {
+                        // Avoid showing a weird error message to the user. See #118179.
```

Comment:
"weird error message" is perhaps less descriptive than it could be. I'd rather have something like "Avoid showing {type error} to users".

Reply:
This is definitely better - thanks!

Reply:
I'll avoid pushing this until I squash, but it now says: `// Avoid showing "{type error}" to users. See #118179.`